cfg/phab_tracker_only.json

```diff
@@ -0,0 +1,127 @@
+{
+  // verbose
+  "simulation": false,
+  "print_timing": false,
+  "use_canvas": true,
+  "use_debug_view": true,  // draw rejected & dropped features on canvas
+  "async_run": false, // turn this off in benchmarking
+
+  // visualization (tracker view) option
+  "draw_OOS": true,
+
+  // memory
+  "memory": {
+    // Values if mapper is not enabled
+    "max_features": 200,
+    "max_groups": 100
```

Comment:
I think you can safely delete `max_groups` from this block, since groups are irrelevant without the filter.